src/mavis/pairing/pairing.py

```diff
@@ -154,9 +154,12 @@ def comparison_distance(
     return max_distance
 
 
-def equivalent(event1: BreakpointPair, event2: BreakpointPair, distances=None) -> bool:
+def equivalent(
+    event1: BreakpointPair, event2: BreakpointPair, distances=None, matching_event_type=True
+) -> bool:
     """
     compares two events by breakpoint position to see if they are equivalent
```

Comment:
should be in an args block like the other docstrings

```
Args:
    - matching_event_type: specificies whether event type must match
```

Reply:
Addressed in [8f1cffb](https://github.com/bcgsc/mavis/pull/336/commits/8f1cffb6e373f828c0cf0c74b796d73bdd3bcd5c)